### PR TITLE
Features/international messages

### PR DESCRIPTION
**Pull request by Sumbal-Tariq**
Cycle 0 ready

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,43 +11,40 @@ gem 'jquery-rails'
 gem 'jquery-ui-rails'
 gem 'mysql2', '~> 0.3.18'
 gem 'nested_form'
+gem 'paperclip'
```
Comment (zainsaleem4338) [inline Gemfile]: version

File: Gemfile [hunk 2]
```diff
@@ -11,43 +11,40 @@ gem 'jquery-rails'
...
-gem 'rspec-rails', '~> 3.8'
 gem 'sass', '~> 3.7', '>= 3.7.4'
 gem 'sass-rails', '~> 5.0'
 gem 'sassc-rails', '>= 2.1.0'
 gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'select2-rails'
 gem 'sequenceid'
 gem 'uglifier', '>= 1.3.0'
+gem 'will_paginate', '~> 3.1.0'
+gem 'will_paginate-bootstrap4'
```
Comment (zainsaleem4338) [inline Gemfile]: version

File: Gemfile [hunk 3]
```diff
@@ -11,43 +11,40 @@ gem 'jquery-rails'
...
 gem 'wicked_pdf', '~>1.4.0'
 gem 'wkhtmltopdf-binary', '~>0.12.4'
 
-group :development do
+group :development, :test do
   gem 'better_errors'
   gem 'capybara', '~> 2.5'
+  gem 'clockwork'
```
Comment (zainsaleem4338) [inline Gemfile]: version

File: Gemfile [hunk 4]
```diff
@@ -11,43 +11,40 @@ gem 'jquery-rails'
...
   gem 'database_cleaner'
+  gem 'delayed_job_active_record'
```
Comment (zainsaleem4338) [inline Gemfile]: version

File: Gemfile [hunk 5]
```diff
@@ -11,43 +11,40 @@ gem 'jquery-rails'
...
   gem 'pry-rails', '=0.3.3'
+  gem 'rails-observers'
   gem 'rspec-rails', '~> 3.8'
   gem 'shoulda-matchers', require: false
   gem 'spring'
   gem 'web-console', '~> 2.0'
 end
 
 group :test do
-  gem 'better_errors'
```
Comment (zainsaleem4338) [inline Gemfile]: why are we removing all these gems?

Comment (Sumbal-Tariq) [reply Gemfile]: development and test common gems move in test development group... thats why..

**Pull request merged**